working/en/index.html

```diff
@@ -0,0 +1,412 @@
+<!DOCTYPE html>
+<html lang="pt-PT">
```

Comment:
This file is inside the "en" folder, then the language can't be "pt-PT"... Check all other files, I know this mistake is in other files.